packages/react-components/react-popover/src/common/isConformant.ts

```diff
@@ -6,7 +6,7 @@ export function isConformant<TProps = {}>(
   testInfo: Omit<IsConformantOptions<TProps>, 'componentPath'> & { componentPath?: string },
 ) {
   const defaultOptions: Partial<IsConformantOptions<TProps>> = {
-    componentPath: module!.parent!.filename.replace('.test', ''),
+    componentPath: require.main?.filename.replace('.test', ''),
```

Comment:
nit: I would prefer to use `require.main!` as you expect that to be always set. The test should rather throw than run with `componentPath: undefined` if `require.main` is not set.

Reply:
using `!`  is a red flag and antipattern ( also not recommended to use by TS team ) . if componenPath will be undefined it will throw anyway